Update devops profile to SRE

pixelwake opened the pull request:
Updates devops to be recently finalized SRE profile

Commit:
Update devops profile to SRE

fureigh commented:
@rooneywp Accurate to assume we'll want any visits to `who-we-are-hiring/positions/devops-engineer/` to be redirected to the SRE URL instead?

Commit:
Very minor copy edits

Made a very very few changes to this document:

- Deleted a few extra spaces
- Added a hyphen 

I'm giving this my thumbs up!

Commit:
Merge pull request #200 from 18F/kategarklavs-patch-14

Very minor copy edits

Commit:
Updated permalink

pixelwake commented:
Merging after final copy proofing.
